fixtures/eurolite/edx-4.json

```diff
@@ -1,11 +1,12 @@
 {
   "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
-  "name": "EDX-4",
+  "name": "EDX-4 4-channel DMX dimmer pack",
+  "shortName": "EDX-4",
   "categories": ["Dimmer"],
   "meta": {
     "authors": ["Paul Rogalski"],
-    "createDate": "2024-10-01",
-    "lastModifyDate": "2024-10-01"
+    "createDate": "2024-10-02",
+    "lastModifyDate": "2024-10-02"
   },
   "links": {
```

Comment:
Could you please replace the links with offical Eurolite ones?

Reply:
Hmm, I am not aware of an official Eurolite website. Could you point me to that please?

Additionally, this dimmer pack isn't sold anymore, so finding links is hard. However there should still be a good amount of them in use (we have it in our small club too).

Reply:
That's the Eurolite website: https://www.steinigke.de/en/Eurolite/

We actually also have this one in our parish youth's event technology team :sweat_smile:

Reply:
If there are no official links anymore, the current ones are fine. Maybe they are available via the Internet Archive?

And also, could you please add a comment about the fixture not being sold anymore?

Reply:
> Maybe they are available via the Internet Archive?

I wasn't able to get an old version, because you need to paste the exact URL into the Wayback Machine.

Reply:
> And also, could you please add a comment about the fixture not being sold anymore?
Done